tools/sklearn/search_model_validation.xml

```diff
@@ -88,6 +89,7 @@ optimizers = getattr(model_selection, optimizers)
 
 options = params["search_schemes"]["options"]
 options['cv'] = get_cv( options['cv'].strip() )
+options['n_jobs'] = int(sys.argv[2])
```

Comment:
try this `options['n_jobs'] = os.environ.get('GALAXY_SLOTS', 1)`

Reply:
The failure was due to package dependency, but your code seems better. 👍